src/config.js

```diff
@@ -81,29 +81,21 @@ import path from 'path';
 /**
  * @typedef {Object} TurboToolsInitConfig
  *
- * @property {AddToPackageJson} addToPackageJson
+ * @property {AddToPackageJson} [addToPackageJson]
  */
 
 /**
  * @typedef {Object} TurboToolsPublishConfig
- * @property {GetCommand} getCommand
- * @property {CheckCanPublish} checkCanPublish
- */
-
-/**
- * @typedef {Object} TurboToolVersionConfig
- * @property {ChangeLogLineFormatter} changelogLineFormatter
+ * @property {GetCommand} [getCommand]
+ * @property {CheckCanPublish} [checkCanPublish]
  */
 
 /**
  * @typedef {Object} TurboToolsConfig
  *
- *
- * @property {TurboToolsInitConfig} init
- *
- * @property {TurboToolsPublishConfig} publish
- *
- * @property {TurboToolVersionConfig} version
+ * @property {TurboToolsInitConfig} [init]
+ * @property {TurboToolsPublishConfig} [publish]
+ * @property {LetsVersionConfig} [letsVersion]
```

Comment:
I think here, it makes more sense for the property to be `version` since that is what maps to the CLI command for turbo-tools. 

That way, if, for whatever reasons, the lets-version gets replaced under the hood here the config is still for the `version` command.

Reply:
Addressed via ed3709048b786cc6817ee82c5d6699c935f6c4f2